src/utils.ts

```diff
@@ -0,0 +1,52 @@
+import extract from 'extract-zip';
+import fs from 'fs-extra';
+import path from 'path';
+import wget from 'wget-improved';
+
+/**
+ * Download repository without using git
+ *
+ * Example:
+ *   url: https://github.com/myOrg/myRepo/archive/main.zip
+ *   output: ~/my/path
+ *
+ * 1. download the repository as a zip to ~/my/path.zip
+ * 2. unzip it to ~/my/path.
+ * 3. delete this zip file
+ *
+ * The issue here is that the zip contains a root folder with the name of the repo
+ * So the files will actually be inside ~/my/path/myRepo
+ *
+ * 4. move all the files up from ~/my/path/myRepo to ~/my/path
+ *
+ * @param url repository url
+ * @param outputPath output path
+ * @returns
+ */
+export async function downloadRepository(url: string, outputPath: string): Promise<void> {
```

Comment:
outputPath is the path where to copy the files no?

Reply:
yep yep